CMakeLists.txt

```diff
@@ -36,10 +45,23 @@ if(BUILD_TESTING)
   ament_add_gtest(test_split test/test_split.cpp)
 
   ament_add_gtest(test_filesystem_helper test/test_filesystem_helper.cpp)
+
+  add_library(toy_test_library SHARED test/toy_test_library.cpp)
```

Comment:
Nitpick: I would suggest to drop the "toy_" prefix everywhere.

Reply:
Done.